gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,31 @@
+id: gigamon
+backend_only: true
+facets:
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_host
+    path: dns_host
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_name
+    path: dns_name
+    source: log
+    type: string
+pipeline:
+  type: pipeline
+  name: gigamon
+  enabled: true
+  filter:
+    query: "source:gigamon"
+  processors: []
```

Comment:
Is it expected that these logs don't have a timestamp field? A disclaimer: without a timestamp field that's mapped with a date remapper, all the logs would be tagged with the ingestion timestamp (when Datadog recieves and processes the logs)